tests/backend/test_design_step_bbox.py

```
@@ -0,0 +1,99 @@
+            status_code=200,
+            content=step_bytes,
+            headers={"Content-Type": "application/step"},
+            request=request,
+        )
+
+    monkeypatch.setattr(httpx.AsyncClient, "request", _mock_request)
+    return expected_dims
+
+
+@pytest.fixture
+def client(cad_service_stub: Dict[str, float]) -> TestClient:
```

Ruff (0.14.1)

Unused function argument: `cad_service_stub`

(ARG001)